packages/ckeditor5-core/src/editor/editorconfig.jsdoc

```diff
@@ -24,7 +24,7 @@
  *			.then( ... )
  *			.catch( ... );
  *
- * Check the {@glink builds/guides/integration/configuration Configuration guide} for more information
+ * Check the {@glink /installation/advanced/predefined-builds Configuration guide} for more information
```

Comment:
Absolute URLs may lead to non-existing resources.

<img width="675" alt="image" src="https://user-images.githubusercontent.com/2270764/155734354-1fc016c0-41e6-4cf0-bdd0-a784cb61903a.png">

<img width="602" alt="image" src="https://user-images.githubusercontent.com/2270764/155734444-42dcb23e-3d54-48b4-a86d-d026d9d4fb63.png">

Reply:
Addressed in [#11422](https://github.com/ckeditor/ckeditor5/pull/11422)